package.json

```diff
@@ -10,47 +10,47 @@
   },
   "dependencies": {
     "@eth-optimism/smock": "^0.2.1-alpha.0",
-    "@openzeppelin/contracts": "3.3.0",
-    "@openzeppelin/contracts-upgradeable": "^3.4.0",
+    "@openzeppelin/contracts": "3.4.1",
+    "@openzeppelin/contracts-upgradeable": "^3.4.1",
     "dotenv": "^8.2.0",
     "hardhat-typechain": "^0.3.5",
-    "openzeppelin-contracts-3.4": "npm:@openzeppelin/contracts@3.4.0",
-    "openzeppelin-contracts-sol-0.5": "npm:@openzeppelin/contracts@2.5.1",
```

Comment:
@penandlim was this pinned to `2.5.1` for any particular reason?

Reply:
This was to compile the timelock contract which was using solidity 0.5.*